src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -5291,6 +5291,7 @@ namespace HeatBalanceManager {
 
 				if ( MaxHeatLoadZone( ZoneNum ) > 1.0e-4 ) { // make sure load big enough to divide
 					MaxHeatLoadZone( ZoneNum ) = std::abs( max( MaxHeatLoadZone( ZoneNum ), MinLoad ) );
+					MaxHeatLoadPrevDay( ZoneNum ) = std::abs( max( MaxHeatLoadPrevDay( ZoneNum ), MinLoad ) );
```

Comment:
@rraustad Well, that's a good point and what you are saying makes sense.

@EnergyArchmage Based on what Rich is saying, is there really a need for this change as you outlined in the GitHub issue?  Is there some other scenario where the previous day's values are not being subjected to the MinLoad comparison already?